src/main/java/com/aws/greengrass/util/NucleusLogsSummarizer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.util;
+
+import java.util.Scanner;
+
+public final class NucleusLogsSummarizer {
```

Comment:
What about Windows log? They already exist and we just need to parse them

Reply:
Updated with windows logging as well